src/main.go

```diff
@@ -642,43 +646,89 @@ func (t *tester) concurrentExecute(querys []query, wg *sync.WaitGroup, errOccure
 	}
 }
 
-func (t *tester) loadQueries() ([]query, error) {
+func (t *tester) LoadQueries() ([]query, error) {
```

Comment:
Seems we don't need to export `loadQueries`, right?

Reply:
Yes, fixed.